src/core_ocean/shared/mpas_ocn_tracer_ecosys.F

```diff
@@ -1188,6 +1190,7 @@ subroutine ocn_tracer_ecosys_surface_flux_compute(activeTracers, ecosysTracers,
          BGC_forcing%SST(column) = activeTracers(indexTemperature,iLevelSurface,iCell)
          BGC_forcing%SSS(column) = activeTracers(indexSalinity,iLevelSurface,iCell)
 
+         ! the 1 indices may need to be changed to iLevelSurface
```

Comment:
@maltrud Do you know whether the 1 index in `ecosysTracers(ecosysIndices%*,1,iCell)` should refer to the surface index (i.e., `minLevelCell`)?

Reply:
The following six lines are a rearrangement of index order:
`(k,iCell,iTracer) <--- (iTracer,k,iCell)`
but only for a single column, and only for the top layer. I'm sure this is for the BGC interface API.

@cbegeman I agree that all 1 indices in the next six lines should be replaced with `iLevelSurface`. I think a shorter index name like `kTop` would be more readable. Perhaps `kSurface`.

Reply:
@mark-petersen Thanks! These changes are in 9d189a2. With this commit, it passes the nightly test suite.